tomb-common/src/types/blockstore/car/carv2/index/indexbucket.rs

```diff
@@ -0,0 +1,11 @@
+use crate::types::streamable::Streamable;
+use std::fmt::Debug;
+use wnfs::libipld::Cid;
+
+/// Special kind of hashmap for dealing with offsets
+pub trait IndexBucket: Debug + Streamable + Send {
```

Comment:
We might want to name this "Indexable" or something of the like to indicate the capability implementing it provides to the bucket.